plugins/bufbuild/protoschema-pubsub/v0.1.0/buf.plugin.yaml

```diff
@@ -0,0 +1,9 @@
+version: v1
+name: buf.build/bufbuild/protoschema-pubsub
+plugin_version: v0.1.0
+source_url: https://github.com/bufbuild/protoschema-plugins
+description: Generates a PubSub schema in the form of a single self-contained messaged normalized to proto2.
+registry:
+  archive: {}
```

Comment:
cc @cyinma can you check the registry config bit?

Reply:
yes, this matched how it looks like in [bufbuild/buf](https://github.com/bufbuild/buf/blob/main/private/bufpkg/bufplugin/bufpluginconfig/testdata/success/archive/buf.plugin.yaml#L8-L9) too